MSAL/test/app/ios/MSALTestAppCacheViewController.m

```diff
@@ -254,82 +248,93 @@ - (void)didReceiveMemoryWarning {
     // Dispose of any resources that can be recreated.
 }
 
-- (void)addTokenToCacheMap:(MSALBaseTokenCacheItem*)item
+- (void)addTokenToCacheMap:(MSALBaseTokenCacheItem *)item
 {
-    NSString* userId = item.user.displayableId;
-    if (!userId)
-    {
-        userId = @"<unknown>";
-    }
+    NSString *userId = item.user.userIdentifier;
     
-    NSMutableDictionary* userTokens = [_cacheMap objectForKey:userId];
+    NSMutableDictionary *userTokens = _cacheMap[userId];
     if (!userTokens)
     {
         userTokens = [NSMutableDictionary new];
         [_cacheMap setObject:userTokens forKey:userId];
     }
     
-    NSString* clientId = item.clientId;
-    NSMutableArray* clientIdTokens = [userTokens objectForKey:clientId];
-    if (!clientIdTokens)
+    NSString *environment;
+    
+    if ([item isKindOfClass:[MSALAccessTokenCacheItem class]])
+    {
+        environment = [NSURL URLWithString:((MSALAccessTokenCacheItem *)item).authority].msalHostWithPort;
+    }
+    else if ([item isKindOfClass:[MSALRefreshTokenCacheItem class]])
     {
-        clientIdTokens = [NSMutableArray new];
-        [userTokens setObject:clientIdTokens forKey:clientId];
+        environment = ((MSALRefreshTokenCacheItem *)item).environment;
```

Comment:
Can we just use item.user.environment here instead for both MSALAccessTokenCacheItem and MSALRefreshTokenCacheItem?